test_data/yam/agama_sle.yaml

```diff
@@ -1,2 +1,15 @@
 ---
 os_release_name: SLES
+repos:
+  - name: SLE-Product-SLES-16.0
+    alias: SUSE_Linux_Enterprise_Server_16.0_x86_64:SLE-Product-SLES-16.0
```

Comment:
IIRC, test data could admit interpolation with other openQA variables, for this case %VERSION%